imaging-ingestion-operator/model/common_test.go

```diff
@@ -0,0 +1,203 @@
+/*
+(C) Copyright IBM Corp. 2021
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package model_test
+
+import (
+	"testing"
+
+	"github.com/Alvearie/imaging-ingestion/imaging-ingestion-operator/model"
+	"github.com/stretchr/testify/require"
```

Comment:
Please run `go mod tidy` so that this new dependency will be added to `go.mod` in project's root folder.

Reply:
done